src/tir/ir/sparse.cc

```diff
@@ -275,6 +276,18 @@ SparseBuffer::SparseBuffer(Var data, Array<Axis> axes, DataType dtype, String na
   node->extra_storage = extra_storage;
   node->name = name;
   node->dtype = dtype;
+  if (!default_value) {
+    if (dtype.is_float())
+      node->default_value = PrimExpr(FloatImm(dtype, 0, span));
```

Comment:
You can just call `Cast(dtype, Integer(0))`, and do not need to handle different data types like this.
Other possible data types in deep learning include `float16`, `bfloat16`, `int64` etc. With `Cast(dtype, Integer(0))` you can handle all cases.